index.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-use-before-define */
+const booksContainer = document.querySelector('.books');
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
+
+const addBtn = document.querySelector('.add-btn');
+
+const Books = JSON.parse(localStorage.getItem('books')) || [];
+
+addBtn.addEventListener('click', () => {
+  const titleValue = titleInput.value;
+  const authorValue = authorInput.value;
+  const bookID = Books.length;
+  addBook(titleValue, authorValue, bookID);
+});
+
+displayBooks();
+
+function addBook(title, author, id) {
+  const book = {
+    title,
+    author,
+    ID: id,
+  };
+  Books.push(book);
+  localStorage.setItem('books', JSON.stringify(Books));
+  // eslint-disable-next-line no-use-before-define
+  displayBooks();
```

Comment:
- [ ] I'm highlighting this section because the previous reviewer did tag it for change. YES!, your addition comes with an `id`, but the removal of your author and tile is not consistent. I've tried adding 5 authors, and on attempting to remove the `id` index 2, two instead of three tiles and authors were deleted. 
- Please consider using a filter method for the `remove` book function, to make sure unique elements are removed 👍